src/components/VehicleBid.jsx

```diff
@@ -22,7 +22,7 @@ const VehicleBid = ({bid, vehicle, shown, chooseBid}) => {
         <dt>Estimated delivery time:</dt>
         <dd>{Math.ceil(bid.time_to_dropoff)} minutes</dd>
         <dt>Cost for delivery:</dt>
-        <dd>{bid.price} <img src={currencyImage} className="currency-symbol" alt="DAV"/></dd>
+        <dd>{Math.round(bid.price * 100) / 100} <img src={currencyImage} className="currency-symbol" alt="DAV"/></dd>
```

Comment:
What are the advantages of this over doing `toFixed()` here?

Reply:
When I tried using toFixed( ) the order screen wouldn't load. I don't have a good argument for an advantage. I'm just a react noobie and this was the best solution I could find :)

Reply:
For some reason it was still passed as a string there. Updated here: https://github.com/DAVFoundation/missions/commit/cc795b41fdc759b17d919c551be6fec25741213a